okhttp/src/main/java/okhttp3/internal/ws/RealWebSocket.kt

```diff
@@ -156,6 +176,7 @@ class RealWebSocket(
         try {
           checkUpgradeSuccess(response, exchange)
           streams = exchange!!.newWebSocketStreams()
+          options = WebSocketOptions.parseServerResponse(response)
```

Comment:
I don’t love that this field gets reassigned. Creating two fields for offered and agreed would be a nice improvement I think.